### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-General_Headings and Labels: Headings on the page are not structured properly as per their visual appearance.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#154]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**MAS ID Violated:** MAS1.3.1, MAS2.4.6

**Repro Steps**: 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Server" in the left menu section and Press Enter.
5. Verify that headings on the page are structured properly.

**Actual Result:**
There are two headings on the page: "Web Server" (main heading) and "General" (sub-heading), and both differs in visual appearance, but both has been provided heading level 1(h1). So Screen Reader is reading both the headings as heading level one.

**Expected Result:**
Main heading (Web Server) should be provided the heading level one (h1) and its sub heading (General) should be provided with heading level two(h2). Narrator should also read the same.

**User Impact:**
Screen Reader users will be unable to understand that which is the main heading and which sub headings and they may find it bit difficult to understand the context of the page.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
[Bug 154_Headings are not structured properly.pptx](https://github.com/Microsoft/IIS.WebManager/files/2331134/Bug.154_Headings.are.not.structured.properly.pptx)

**Comment by jhkimnew** [comment]
We have refactored the UI for WAC and its usage is changed and this issue seems not be applicable anymore. We will revisit this issue when there is a real user's feedback.

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
